src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -138,6 +139,148 @@ void RotatedSPOs::resetParametersExclusive(const opt_variables_type& active)
   }
 }
 
+void RotatedSPOs::writeVariationalParameters(hdf_archive& hout)
+{
+  hout.push("RotatedSPOs");
+  if (use_global_rot_)
+  {
+    hid_t grp                   = hout.push("rotation_global");
+    std::string rot_global_name = std::string("rotation_global_") + SPOSet::getName();
```

Comment:
Could you add const if the variable won't be modified?

Reply:
We should have a separate discussion about what good ideas we want to enforce on the code.   Adding 'const' to variables is not listed in the "Uses of const" section of the coding standards.   And I question whether it's necessary for variables - it adds extra characters to the declaration without adding a lot of information (IMO).